presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java

```diff
@@ -190,6 +203,24 @@ public void testDuplicatedRowCreateTable()
                 "line 1:49: Column name 'orderkey' specified more than once");
     }
 
+    @Test
+    public void testCreateTableAndShow()
+    {
+        checkCreateTableSupported();
+        try {
+            String catalog = getSession().getCatalog().get();
+            assertUpdate("CREATE TABLE test_show_create (" +
+                    "column_a DATE" +
+                    ")");
+            assertQuery("SHOW CREATE TABLE tpch.test_show_create", "VALUES 'CREATE TABLE " + catalog + ".tpch.test_show_create (\n" +
```

Comment:
formatting here is a bit odd... maybe put each argument in separate line, then spliting second argument should look nicer